### DISCUSSION TITLE
关于我

### DISCUSSION THREAD

**Discussion by giscus** [keelii/fresh-blog#6] [Announcements]
# 关于我

芝兰生于深林，不以无人而不芳，君子修道立德，不为穷困而败节。

https://keelii.com/about

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。

**Reply by keelii** [reply]
我没兴趣，请不要将我的文章以任何形式转到腾讯平台。

**Comment by venomwise** [comment]
大佬，你的Awesome Host Manager插件被Google下架了，没法在chrome的插件市场下载，方便提供个下载地址吗？

**Reply by keelii** [reply]
没下架呀？https://chromewebstore.google.com/detail/awesome-host-manager/pikaoeecieigblebdddckmlegonlogha
安装不了的话可以上代码库里面下载awesome-host-manager.crx文件拖到浏览器里面：https://github.com/keelii/awesome-host-manager

**Reply by venomwise** [reply]
通过crx安装了，但是没法启用，有啥办法吗大佬

**Reply by keelii** [reply]
好像确实不行了，我原来是chrome store上安装的还能用。回头我找时间看下能不能再发一版。

**Reply by SimonGH0STRiley** [reply]
好想在chrome应用商店上有个新版，但是新版没有之前的点击下拉弹窗

**Reply by keelii** [reply]
那个版本有问题，稍后我看下。

**Reply by keelii** [reply]
@SimonGH0STRiley 新版已发布，可以通过官方下载使用：https://chromewebstore.google.com/detail/awesome-host-manager/nmphnmocfajfalkembmgopadlkjjhend

**Reply by SimonGH0STRiley** [reply]
收到，已可正常使用。感谢作者的付出
